client/my-sites/hosting/sftp-card/index.js

```diff
@@ -136,34 +135,42 @@ const SftpCard = ( {
 			<CardHeading>{ translate( 'SFTP Credentials' ) }</CardHeading>
 			<div className="sftp-card__body">
 				<p>
-					{ translate( "Access and edit your website's files directly by using an SFTP client." ) }
+					{ username
+						? translate(
+								'Use the credentials below to access and edit your website files using an SFTP client. {{a}}Learn more about SFTP on WordPress.com{{/a}}.',
+								{
+									components: {
+										a: (
+											<ExternalLink
+												icon
+												target="_blank"
+												href={ localizeUrl( 'https://en.support.wordpress.com/sftp/' ) }
+											/>
+										),
+									},
+								}
+						  )
+						: translate(
+								"Access and edit your website's files directly by creating SFTP credentials and using an SFTP client."
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Access and edit your website's files directly by using an SFTP client.' )` *ES Score: 7*